source/sdk/structs/Stack.ooc

```diff
@@ -23,17 +23,16 @@ Stack: class <T> {
 				Exception new(This, "Trying to pop an empty stack.") throw()
 		this _data[--this _size]
 	}
+	top: func -> T {
```

Comment:
I don't know, you can decide.
It *will* be a function under the hood, anyway :)